setup.py

```diff
@@ -1,11 +1,20 @@
 from distutils.core import setup, Extension
-
-module1 = Extension('ale_python_interface.ale_c_wrapper',
-                    libraries = ['ale_c'],
-                    include_dirs = ['src'],
-                    library_dirs = ['ale_python_interface'],
-                    extra_compile_args=['-D__STDC_CONSTANT_MACROS'],
-                    sources=['ale_python_interface/ale_c_wrapper.cpp'])
+import sys
+if sys.platform == 'darwin':
+    module1 = Extension('ale_python_interface.ale_c_wrapper',
+                        libraries = ['ale_c'],
+                        include_dirs = ['src','/usr/local/include/SDL'],
```

Comment:
I didn't have issues compiling the Python interface on Mac OS X. These seem like reasonable flags (except /usr/local/include/SDL is user-specific), but I wouldn't change the setting unless this is better for all users. @mhauskn?